app/lib/brexit_checker/notifications.yaml

```diff
@@ -23,12 +23,18 @@ notifications:
         - visiting-eu
   # End of example block.
   # Add new notifications below 👇
-  - uuid: 6e18b271-c4c2-4937-8949-d587453801df
-    type: addition
-    action_id: T131
-    date: '2021-02-04'
-  - uuid: 19d40094-76c2-4e40-a1fb-44dc7fa76a47
+  - uuid: 2bd71314-e9d2-4b0d-92a4-872620281401
+    type: content_change
+    action_id: T025
+    date: '2021-02-23'
+    note: Added information about new rules not applying to Northern Ireland
+  - uuid: 03c4f7e7-99bb-4dc2-aa2a-4438b8d6aff5
+    type: content_change
+    action_id: T071
+    date: '2021-02-23'
+    note: Lead time updated to to 4 days
```

Comment:
There's a typo here (and in the trello card!) too many to's

Reply:
My weary, weary eyes. Thank you